File: Java/Program/Shirt.java
public class Shirt extends Clothing {
    private String color;
    private String sleeveType;

    public Shirt(String name, String brand, int price, String size, String material, String gender, String color, String sleeveType) {
        super(name, brand, price, size, material, gender);
        this.color = color;
        this.sleeveType = sleeveType;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getSleeveType() {
        return sleeveType;
    }

    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }
}

File: CPP/Program/Shirt.cpp
#include <iostream>
#include <string>
#include "Clothing.cpp"

class Shirt : public Clothing {
private:
    std::string color;
    std::string sleeveType;

public:
    Shirt(std::string name, std::string brand, int price, std::string size, std::string material, std::string gender, std::string color, std::string sleeveType)
        : Clothing(name, brand, price, size, material, gender) {
        this->color = color;
        this->sleeveType = sleeveType;
    }

    std::string getColor() {
        return color;
    }

    void setColor(std::string color) {
        this->color = color;
    }

    std::string getSleeveType() {
        return sleeveType;
    }

    void setSleeveType(std::string sleeveType) {
        this->sleeveType = sleeveType;
    }
};
